app/src/main/java/com/andrew/tasky/agenda/presentation/screens/event_detail/EventDetailViewModel.kt

```diff
@@ -95,6 +102,17 @@ class EventDetailViewModel @Inject constructor(
         _photo.value = photoList
     }
 
+    val uiEventPhotos = photos.map { photos ->
+        photos.map { photo ->
+            UiEventPhoto.Photo(photo)
+        }.toMutableList<UiEventPhoto>()
+            .apply {
+                if ((size in 1..9) && (!isAttendee.value)) {
```

Comment:
Avoid calling `.value` in a flow operator like here. That's very vulnerable to race conditions. In this case it's unlikely, but imagine `isAttendee` is a frequently changing flow. You could jump into this flow block here, read the first value of `isAttendee.value` and if it then changes right after the flow block will still execute with the previously read value. Instead you would need to use combine:
```
val uiEventPhotos = photos.combine(isAttendee) { photos, isAttendee ->
    // here you can now access isAttendee safely since it will be retriggered on every change
}
```